### ISSUE TITLE
Lines over screen [Tales of Legendia] [software mode]

### ISSUE THREAD

**Issue by clbr** [PCSX2/pcsx2#1421]
In pcsx2 1.4.0, in gdsx software mode, Tales of Legendia has lines all over the screen. Google screenshots do not have those, so it must be a pcsx2 bug.

![legendia](https://cloud.githubusercontent.com/assets/790333/16176928/d5a0c8f0-360c-11e6-8d77-57065e6ae0da.png)

**Comment by gregory38** [comment]
TV shader ?

**Comment by FlatOutPS2** [comment]
> Google screenshots do not have those, so it must be a pcsx2 bug.

Seriously? :laughing: 
It could be a bug, but just because a google screenshot doesn't show something doesn't mean it's not supposed to be there.

**Comment by vsub** [comment]
Yes,this does seems to exist in software mode only...it exist even on GSdx from 05.05.2015

**Comment by clbr** [comment]
There's been complaints about this in the pcsx2 forum going back to 2011 at least.

**Comment by ramapcsx2** [comment]
Yea, software rendering has a problem with this. I think this is one of these channel shuffle effects by the way.

**Comment by gregory38** [comment]
Could someone test the rendering in 1.4 and latest git with openGL HW, no crc, native, accurate blending to full.

**Comment by orbea** [comment]
OS: `Slackware64-current`
`pcsx2-36dd500_2016.06.23_master-x86_64-1_git`

openGL HW, no crc, native, accurate blending to full
![1](http://ks392457.kimsufi.com/orbea/stuff/pics/scrots/pcsx2/2016-06-25-114058_1680x1050_scrot.png)

SW
![2](http://ks392457.kimsufi.com/orbea/stuff/pics/scrots/pcsx2/2016-06-25-114202_1680x1050_scrot.png)

**Comment by gregory38** [comment]
@orbea do you have a screenshot of the 1.4 (HW only)?

**Comment by orbea** [comment]
`pcsx2-1.4.0-x86_64-1_SBo1`

openGL HW, no crc, native, accurate blending to full
![3](http://ks392457.kimsufi.com/orbea/stuff/pics/scrots/pcsx2/2016-06-26-165106_1680x1050_scrot.png)

**Comment by gregory38** [comment]
Could someone post a GSdump of the issue. I want to see if I can reproduce it on my side.

**Comment by vsub** [comment]
Here(change the extension to zip)
![snaps](https://cloud.githubusercontent.com/assets/5032540/17747191/592ed2b0-64bc-11e6-9b64-d216494da8c3.jpg)

**Comment by gregory38** [comment]
Hum, I have a massive blur when I enable accurate blending. I suspect the game to do a kind of blur based on the depth. Rendering is quite complex (lots of processing effect) so it will take me some time to debug it further.

**Comment by gregory38** [comment]
Hum, fwiw, I have the line on the HW renderer too when I enable accurate blending. However I see them in the middle of the rendering, not on the final frame which got a massive blur.

**Comment by seta-san** [comment]
can confirm that this is still happening 1.7.2631

**Comment by Mrlinkwii** [comment]
updated GS dump
[issue _1421.zip](https://github.com/PCSX2/pcsx2/files/10677148/issue._1421.zip)

**Comment by seta-san** [comment]
still happening on 1.7.4920

**Comment by lightningterror** [comment]
If you enable CPU sprite render size to 8 and set blended sprites the lines are also present on hw renderers.

**Comment by ScottNash042** [comment]
still happening on v2.7.10
